### PR TITLE
feature(unlock-app): Event collections' settings page

### PR DESCRIPTION
**Pull request by 0xTxbi**
# Description
This PR introduces a settings page for an event collection.

## ScreenGrab
Screenshot 2024-09-24 at 15 45 57

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/app/events/[slug]/settings/page.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { AuthRequired } from 'app/Components/ProtectedContent'
+import { Metadata } from 'next'
+import { notFound } from 'next/navigation'
+import { EventCollectionSettings } from '~/components/content/events-collection/EventCollectionSettings'
+import { getEventCollection } from '~/utils/eventCollections'
+
+export const metadata: Metadata = {
+  title: 'Event Settings',
```
Comment (julien51) [inline unlock-app/app/events/[slug]/settings/page.tsx:8]:
```suggestion
  title: 'Events Collection Settings',
```

File: unlock-app/app/events/[slug]/settings/page.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  description: 'Manage your event settings.',
```
Comment (julien51) [inline unlock-app/app/events/[slug]/settings/page.tsx:9]:
```suggestion
  description: 'Manage your collection settings.',
```

**Review by julien51 — APPROVED** [review]

File: unlock-app/app/events/[slug]/settings/page.tsx (hunk 1 above)
Comment (0xTxbi) [reply unlock-app/app/events/[slug]/settings/page.tsx]: an oversight, thank you!

File: unlock-app/app/events/[slug]/settings/page.tsx (hunk 2 above)
Comment (0xTxbi) [reply unlock-app/app/events/[slug]/settings/page.tsx:9]: another one, thanks!

**Comment by julien51** [comment]
ok LGTM, feel free to merge once it passes!

**Pull request merged**